main.js

```diff
@@ -96,7 +96,7 @@ function initMainWindow() {
       webviewTag: false,
       nodeIntegration: false,
       sandbox: true,
-      contextIsolation: true,
+      contextIsolation: false,
```

Comment:
For some reason contextIsolation: true will ruin attaching things to the window from the preloader.